coordinator/manifest/manifest.go

```diff
@@ -338,6 +342,26 @@ func (m Manifest) Check(ctx context.Context, zaplogger *zap.Logger) error {
 		}
 	}
 
+	seenNames := make(map[string]bool)
+	for _, srv := range m.TimeServers {
+		if _, ok := seenNames[srv.Name]; ok {
+			return fmt.Errorf("duplicated server name in Roughtime config: %q", srv.Name)
+		}
+		seenNames[srv.Name] = true
+
+		if srv.PublicKeyType != "ed25519" {
+			return fmt.Errorf("invalid public key type %q specified for Roughtime server %q", srv.PublicKeyType, srv.Name)
+		}
+
+		udpAddr, err := serverUDPAddr(&srv)
+		if err != nil {
+			return fmt.Errorf("Roughtime server %q lists invalid address: %s", srv.Name, err)
```

Comment:
```suggestion
			return fmt.Errorf("roughtime server %q lists invalid address: %s", srv.Name, err)
```
error strings should not be capitalized

Reply:
Roughtime is a proper noun, which is[ capitalized even in error strings](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).